### PR TITLE
update changelog and publish new snapshot

### PR DESCRIPTION
**Pull request by aaronpk**

### CODE REVIEW

File: public/source/index.php [hunk 1]
```diff
@@ -856,8 +856,9 @@
       <section>
         <h3>Changes from 26 September 2020 to this version</h3>
         <ul>
-          <li>Remove same-domain requirement for entered and final profile URL by instead confirming the authorization server</li>
+          <li>Remove same-domain requirement for entered and final profile URL by instead having the client confirm the authorization server</li>
```
Comment (sknebel) [inline public/source/index.php:859]: minor usability: link the section?

File: public/source/index.php [hunk 2]
```diff
@@ -856,8 +856,9 @@
...
           <li>Only the <code>me</code> value returned by the authorization server is a profile URL, do not refer to the user provided URL as such</li>
+          <li>Editorial changes to use the term "hostname" instead of "domain" where appropriate</li>
```
Comment (Zegnat) [inline public/source/index.php:861]: Did we borrow the term “hostname” from somewhere? URL spec uses only “[host](https://url.spec.whatwg.org/#hosts-(domains-and-ip-addresses))”. `hostname` in the URL spec only exists as a `USVString` attribute on the URL class.

Comment (aaronpk) [reply public/source/index.php]: Fair enough! I've replaced these with "host" instead.

File: public/source/index.php (hunk 1 above)
Comment (aaronpk) [reply public/source/index.php]: done

**Review by sknebel — APPROVED** [review]

**Pull request merged**